test/csuite/schema_abort/main.c

```diff
@@ -620,8 +621,14 @@ thread_ckpt_run(void *arg)
             }
         }
 
-        /* Set the configurations. Set use_timestamps regardless of whether timestamps are in use.
```

Comment:
I suspect this comment is left over somehow - it does not appear to apply to the `session->checkpoint` line. So removing it would be the right thing to do. Just thought that was worth confirming.